about.html

```diff
@@ -0,0 +1,150 @@
+                About
+            </li></a>
+           <a href="#program"> <li class="navigationItem">
+                Program
+            </li></a>
+            <a href="#join"> <li class="navigationItem">
+                Join
+            </li></a>
+           <a href="#sponsor">
+           <a href="#sponsor"> <li class="navigationItem">
+                Sponsor
+            </li>
+        </a>
+        </a>
+           <a href="#news"> <li class="navigationItem">
+                News
+            </li>
+        </a>
+        </ul>
+    </div>
+    <header class="headerAbout flex">
+        <h1 class="mainTitle">
+            "Mellon!""
+        </h1>
+        <h2 class="secondTitle">
+            ARE YOU READY  A JOURNEY TO  MIDDLE-EARTH?
+        </h2>
+    </header>
+    <main>
+    <div class="indexAbout flex" id="about">
+        <p class="description">
+            We meet every year on Tolkien's birthday date on 3 January and give our respect to his memory. It is a good opportunity to meet new people who are similar to you. Also, the legacy of Tolkien is lived by the way.  
+        </p>
+        <p class="indexDescription">
+            Please contact us per Email for any further questions about Mellor!</p>
+        <p class="email">
+burak.otman@bpdesign.com.tr
+        </p>
+    </div>
+    <div class="commonLogo flex">
+        <h2 class="logoTitle">
+            bp Organization <br> Mellor's Logo
+        </h2>
+        <p class="logoDescription">
+            The logo of Mellor was decided by coming from votes of Tolkien fans from 1. September to 1. November in 2022. 
+        </p>
+        <img src="./mellonLogo.png" class="mellorLogo" alt="mellorLogo">
+    </div>
+    <div class="pastEvents flex" id="news">
+<h2 class="pastEventsTitle">
+    See the past Mellon Events
+</h2>
+<p class="pastEventsDescription">
+    Take a look at last two Mellon Events which were in Simav and Zonguldak in Turkey.
+</p>
+<div class="eventImage flex">
+<div class="golcuk flex">
+    <div class="imageLayout">
+        <div class="eventIndex">
+    <h2 class="eventTitle">
+        2021
+    </h2>
+    <p class="eventDescription">
+        Mellon 2021 in Simav
+    </p>
+    </div>
+</div>
+</div>
+<div class="zonguldak flex">
+    <div class="imageLayout">
+        <div class="eventIndex">
+    <h2 class="eventTitle">
+        2022
+    </h2>
+    <p class="eventDescription">
+        Mellon 2022 in Zonguldak
+    </p>
+</div>
+</div>
+</div>
+</div>
+    </div>
+    <div class="partner" id="sponsors">
+        <h2 class="partnerTitle">
+            Partner
+        </h2>
+        <div class="partnersList flex">
+<img class="partnerImage" src="dost.png"><img class="partnerImage" src="reuters.png"><img class="partnerImage" src="hek.jpg"><img class="partnerImage" src="Warner_Bros._(2019)_logo.svg.png"><img class="partnerImage" src="simav-belediyesi-og.jpg">
+        </div>
+        
+    </div>
```

Comment:
- [ ] Kindly ensure that the partners section in the **Mobile view** scales the whole width of the page as it is in the Behance design.

**This is how your design looks like now:**

![image](https://user-images.githubusercontent.com/86318284/212130004-2108db43-7b47-4f62-922b-afc423028300.png)


**This is the desired design:**

![image](https://user-images.githubusercontent.com/86318284/212130107-a0d4fb72-81de-43c0-9264-a24f840091ac.png)